javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,46 @@
+var $allKeys = document.querySelectorAll('span');
+var $accuracy = document.querySelector('.accuracy');
+var $playAgain = document.querySelector('.play-again');
+var index = 0;
+var nextKey = $allKeys[index];
+var missedCount = 0;
+
+function pressKey(event) {
+  var isNbspKey = false; // Must check because &nbsp; !== ' '
+  if (nextKey.textContent.charCodeAt(0) === 160) {
+    isNbspKey = true;
+  }
```

Comment:
You can make this more concise since comparisons always evaluate to a Boolean.

```js
var isNbspKey = nextKey.textContent.charCodeAt(0) === 160;
```

Reply:
Oh cool, I didn't realize I could do that! And I am now remembering that you posted a link to Cockos yesterday but I didn't realize that was for this assignment, so I will go and figure out how to do that 😄